pkg/volume/scaleio/sio_volume_test.go

```diff
@@ -425,8 +426,8 @@ func TestVolumeProvisionerWithIncompleteConfig(t *testing.T) {
 	}
 }
 
-func TestVolumeProvisionerWithZeroCapacity(t *testing.T) {
-	plugMgr, tmpDir := newPluginMgr(t, makeScaleIOSecret(testSecret, testns))
+func TestVolumeProvisionerWithSecretNamespace(t *testing.T) {
```

Comment:
This change seems unrelated? Maybe pulled in from another branch by accident?

Reply:
hmm weird. It does. I fixed it.